src/ProjectTemplates/Web.ItemTemplates/content/RazorView/.template.config/template.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [ "Web", "ASP.NET" ],
+  "name": "Razor View",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "Am empty razor view",
+  "tags": {
+    "language": "C#",
+    "type": "item"
+  },
+  "groupIdentity": "Microsoft.AspNetCore.Mvc.RazorView",
+  "precedence": "9801",
+  "identity": "Microsoft.AspNetCore.Mvc.RazorView.7.0",
```

Comment:
shouldn't this be 8.0 to match the other templates in main?

Reply:
fixed.